app/services/statements/query.rb

```diff
@@ -1,19 +1,39 @@
 module Statements
   class Query
-    def initialize(lead_provider:)
-      @lead_provider = lead_provider
+    def initialize
+      @scope = Statement.all
+    end
+
+    def by_lead_provider(lead_provider)
+      @scope = @scope
+                 .includes(:cohort)
+                 .where(lead_provider:)
+
+      self
+    end
+
+    def by_cohorts(*start_years)
```

Comment:
`#with_cohort_start_years`? 🤔

Reply:
Yeah - the `by_` makes it sound like it's 'grouped by' them rather than limited to them. Naming is always the hardest bit!

Reply:
`with_cohorts` maybe?